mlflow/projects/backend/local.py

```diff
@@ -360,9 +365,9 @@ def _get_hdfs_artifact_cmd_and_envs(artifact_repo):
     # pylint: disable=unused-argument
     cmds = []
     envs = {
-        "MLFLOW_KERBEROS_TICKET_CACHE": os.environ.get("MLFLOW_KERBEROS_TICKET_CACHE"),
-        "MLFLOW_KERBEROS_USER": os.environ.get("MLFLOW_KERBEROS_USER"),
-        "MLFLOW_PYARROW_EXTRA_CONF": os.environ.get("MLFLOW_PYARROW_EXTRA_CONF"),
+        "MLFLOW_KERBEROS_TICKET_CACHE": MLFLOW_KERBEROS_TICKET_CACHE.get(),
```

Comment:
Can we also update references to these environment variables in https://github.com/mlflow/mlflow/blob/91899ac0f907da34ce0e8b952dc69db6a2f7da23/mlflow/store/artifact/hdfs_artifact_repo.py#L172-L174?